example/distill/nlp/README.md

```diff
@@ -0,0 +1,50 @@
+# ERNIE distillation
+
+## Dataset
+Chinese sentiment dataset
+
+## Local test
+## Finetune or downlod finetuned model
+### Download dataset
+```
+wget https://paddle-edl.bj.bcebos.com/distillation/chnsenticorp/data.tgz
+tar -xzvf data.tgz
+```
+
+### Finetune
+```
+nohup python -u ./fine_tune.py > finetune.log 2>&1 &
+```
+
+When the finetune job completes, the directory `ernie_senti_server` and `ernie_senti_client` will be generated.
+ 
+### Download finetuned model
```

Comment:
Download the teacher model

Reply:
you would like to tell user how to distill the knowledge of a teacher to a student in this example, so you'd better not involve new concept about finetuning.

Reply:
Done.